src/document.rs

```diff
@@ -243,20 +227,20 @@ fn value_to_py(py: Python, value: &Value) -> PyResult<PyObject> {
             .into_py(py)
         }
         Value::Facet(f) => Facet { inner: f.clone() }.into_py(py),
-        Value::JsonObject(json_object) => {
-            let inner: HashMap<_, _> = json_object
-                .iter()
-                .map(|(k, v)| (k, value_to_object(v, py)))
-                .collect();
+        Value::Array(arr) => {
+            let inner: Vec<_> =
+                arr.iter().map(|x| value_to_py(py, x)).collect::<PyResult<_>>()?;
             inner.to_object(py)
-        }
+        },
+        Value::Object(obj) => object_to_py(py, obj)?,
         Value::Bool(b) => b.into_py(py),
         Value::IpAddr(i) => (*i).to_string().into_py(py),
     })
 }
 
 fn value_to_string(value: &Value) -> String {
     match value {
+        Value::Null => "null".to_string(),
```

Comment:
Is there a better string here?

Reply:
Maybe "None", but I am also wondering if returning an empty string "" is possible, because it may be used in boolean expressions in python ?

Reply:
I think you're right. An empty string is the closest thing to null while retaining a string type. It will look the same as a "real" empty string, but I don't think there's much we can here given that the result type of this function is `String`.

Reply:
I decided to rather go with the debug formatter of `OwnedValue` here:

```rust
        Value::Null => format!("{:?}", value),
```

This is because `value_to_string()` is currently used only in the debug formatter for `Document`.